shared-helpers/src/auth/AuthContext.ts

```diff
@@ -223,16 +224,25 @@ export const AuthProvider: FunctionComponent<React.PropsWithChildren> = ({ child
       email,
       password,
       mfaCode: string | undefined = undefined,
-      mfaType: MfaType | undefined = undefined
+      mfaType: MfaType | undefined = undefined,
+      forPartners: boolean | undefined = undefined
```

Comment:
Why do we need this forPartners piece?

Reply:
the backend doesn't really know if the user logging in is logging into public or partners

so where we do the login calls we make sure the user is able to login to partners